tools/c7n_gcp/c7n_gcp/resources/sql.py

```diff
@@ -159,11 +154,28 @@ class resource_type(ChildTypeInfo):
         }
 
         @staticmethod
-        def get(client, resource_info):
-            parameters = SqlInstanceChildWithSelfLink._get_base_query_parameters(resource_info)
-            parameters['id'] = resource_info['backup_run_id']
+        def get(client, event):
+            project = jmespath.search('protoPayload.response.targetProject', event)
+            instance = jmespath.search('protoPayload.response.targetId', event)
+            insert_time = jmespath.search('protoPayload.response.insertTime', event)
+            parameters = {'project': project,
+                          'instance': instance,
+                          'id': SqlBackupRun.resource_type.from_insert_time_to_id(insert_time)}
             return client.execute_command('get', parameters)
 
+        @staticmethod
+        def from_insert_time_to_id(insert_time):
```

Comment:
this method should be private prefixed with `_` .. I'd also like to see some unit tests for this independently of its use in `get`

Reply:
Made the method private and added a test.